app.js

```diff
@@ -94,6 +102,34 @@ function encryptProverb() {
 
 function letterCheck(clickedLetter) {
   state.roundCounter++;
-  (state.proverb.includes(clickedLetter) === true) ? (state.correct.push(clickedLetter)) : (state.wrong.push(clickedLetter));
+  (state.proverb.includes(clickedLetter)) ? (correctGuess(clickedLetter)) : (wrongGuess(clickedLetter));
   displayProgress();
 }
+
+function correctGuess(letter) {
+  state.correct.push(letter);
+  const clickedLetter = document.querySelector('.' + letter);
+  clickedLetter.classList.add('clicked-correct');
+}
+
+function wrongGuess(letter) {
+  state.wrong.push(letter);
+  const clickedLetter = document.querySelector('.' + letter);
+  clickedLetter.classList.add('clicked-wrong');
+  insertNextImg();
+  if (state.wrong.length === 9) {
+    gameOver();
+  }
+}
+
+function gameOver() {
+  mainBtn.innerHTML = 'PLAY AGAIN?';
+  mainBtn.removeEventListener('click', gameSetup);
```

Comment:
better would be only one function -> in example: `startGame` or something like this, which invoke `gameSetup` or `restartGame`, based on information if game is starting for the first time or is restarting.